section2/reflection.md

```diff
@@ -1,15 +1,30 @@
 ## Section 2 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+`The SMART acronym was a new way of presenting something I have encountered before. Not being afraid to make mistakes; I am not afraid to make mistakes on my own, but to share them out to the people around me is difficult.`
```

Comment:
Yep, I can understand that! It is definitely uncomfortable. And... you need to make those mistakes to learn/it's just part of being a developer. Keep yourself aware of it, and keep an eye out for when you see your peers do it, and how much they/others around them benefit from doing so.